### ISSUE TITLE
No module named: 'aries_cloudagent.core.profile'

### ISSUE THREAD

**Issue by blu3beri** [hyperledger/aries-acapy-plugin-toolbox#48]
After installing the plugin toolbox according to the README and running this command:

```
aca-py start \                                                                                                                                                        (update)
              -it http localhost 3000 -it ws localhost 3001 \
              -ot http \
              -e http://localhost:3000 ws://localhost:3001 \
              --plugin acapy_plugin_toolbox \ 
              --no-ledger
```
I get the following error:

```2020-12-30 12:54:34,568 aries_cloudagent.core.plugin_registry ERROR Error loading plugin module: Unable to import module acapy_plugin_toolbox: No module named 'aries_cloudagent.core.profile'``

Is there anything I am missing or is there a way to fix this quickly?

**Comment by dbluhm** [comment]
This appears to be a result of a version mismatch between the plugin and ACA-Py. What version of ACA-Py are you using? The plugin at master HEAD is currently in a pre-release state in anticipation of ACA-Py 0.6.0 and targets a commit of ACA-Py rather than a specific version. Once 0.6.0 is released, an official release of the plugin targeting that version will also be released.

**Comment by blu3beri** [comment]
The aca-py version is ```0.5.6```.
The Aries-plugin-toolbox version is ```0.1.0``` I think, it is the most recent one.

Which versions should I use for both of them to start working again? (the latest working version, that is)

**Comment by blu3beri** [comment]
I fixed it. 

- aca-py version ```0.5.6```
- aries-acapy-plugin-toolbox from [my fork](https://github.com/blu3beri/aries-acapy-plugin-toolbox) it is based on a commit from the main repo. 

Until the latest versions match again this should work if people have the same issue.

**Comment by dbluhm** [comment]
Glad you've worked things out and apologies for the limited support for versions between 0.5.3 and 0.6.0-pre. Development time has been focused elsewhere but hopefully this will be resolved shortly with an official release of 0.6.0. If you have reason to continue with 0.5.6 after the 0.6.0 release, I'd be happy to create a branch and resolve conflicts to create a tagged release for 0.5.6 with the stipulation that some toolbox functionality will not be available for that branch.
